### ISSUE TITLE
Consider a release that is compiled with Java 7 for legacy projects

### ISSUE THREAD

**Issue by dcominottim** [radcortez/openjpa-gradle-plugin#2]
@radcortez, is there any chance of a release of this plugin compiled with Java 7? I am working on several legacy projects that need some ugly workarounds in Gradle (forks for compiler JVMs) and that both JDKs (8 and 7) be installed in order for the build to work. Otherwise, I might need to work with a fork of the code to simplify things on the projects' side. :/

**Comment by radcortez** [comment]
hey @dcominottim, let me have a look. I think it should be ok to release with 7 compatibility. Please allow me a few days.

**Comment by radcortez** [comment]
Btw, would you mind telling me what is the exact issue that you are getting on the build? Exception stacktrace, etc?

**Comment by radcortez** [comment]
@dcominottim ?

**Comment by dcominottim** [comment]
Sorry, @radcortez, I had to take care of some urgent matters and couldn't answer earlier.

I suppose that Gradle pulls plugins from Bintray in JAR format, is that it? If that is the case, I suppose the problem is that if the JAR was compiled with Java 8, it won't work when Java 7 is used to run Gradle. I will confirm the exact error message in a couple hours.

**Comment by dcominottim** [comment]
Here is the console output when JAVA_HOME is set to JDK 7 instead of JDK 8:

```
A problem occurred evaluating root project 'NEAS'.
> java.lang.UnsupportedClassVersionError: com/radcortez/gradle/plugin/openjpa/Op
enJpaPlugin : Unsupported major.minor version 52.0

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug
option to get more log output.
```

Please let me know if you need more info.

**Comment by radcortez** [comment]
@dcominottim thanks. Ill see what I can do.

**Comment by radcortez** [comment]
Hi @dcominottim 

Sorry for taking some time, was extremely busy. I've just released version 1.2 with JDK 7 support. You can grab it here:
https://bintray.com/radcortez/radcortez/openjpa-gradle-plugin#

Let me know if it fixes your problem.

Cheers,
